lib/Router.dart

```diff
@@ -82,6 +82,10 @@ class Router {
     Navigator.of(context).pop();
   }
 
+  static void formComplete(BuildContext context) {
+    Navigator.of(context).pushNamedAndRemoveUntil('/home', ModalRoute.withName('/login'));
+  }
```

Comment:
Can't test this. When trying to select pdf app crashes. Checked with two phones.